### PR TITLE
Wagtail 3.0

### PR DESCRIPTION
**Pull request by BrianXu20**
Ticket: https://projects.torchbox.com/projects/support-team/tickets/510

Support Wagtail 3.0

CI: https://github.com/BrianXu20/wagtailguide/pull/2

Screenshot: 
Screen Shot 2022-06-15 at 3 48 49 PM

### CODE REVIEW

File: wagtail_guide/tests/tests.py [hunk 1]
```diff
@@ -2,10 +2,7 @@
 from django.contrib.auth.models import Group
 from django.test import TestCase
 from django.urls import reverse
-from wagtail.core.models import Page
-from wagtail.tests.utils import WagtailTestUtils
-from wagtail.tests.utils.form_data import (inline_formset, nested_form_data,
-                                           streamfield)
+from wagtail.test.utils import WagtailTestUtils
```
Comment (nickmoreton) [inline wagtail_guide/tests/tests.py]: **Questions**: Does this need the `from wagtail import VERSION as WAGTAIL_VERSION` check here

**Review by nickmoreton — COMMENTED** [review]
All looks good to me, just one question.

File: wagtail_guide/tests/tests.py (hunk 1 above)
Comment (BrianXu20) [reply wagtail_guide/tests/tests.py]: I don't think so, those imports are not being used, so I removed them.

Comment (nickmoreton) [reply wagtail_guide/tests/tests.py]: Sorry I don't think  I was clear with my question there.

In the upgrade docs: https://docs.wagtail.org/en/stable/releases/3.0.html#changes-to-module-paths doesn't the `from wagtail.test.utils import WagtailTestUtils` need to be conditional for the version?

E.G.
> The wagtail.tests module is renamed to wagtail.test

so we can support v2.16

Comment (BrianXu20) [reply wagtail_guide/tests/tests.py]: You are right, good catch! I will update this soon
Screen Shot 2022-06-15 at 5 17 07 PM

Comment (BrianXu20) [reply wagtail_guide/tests/tests.py]: Fixed in https://github.com/kevinhowbrook/wagtailguide/pull/19/commits/db1d83f912809823e1252d3acc89cd89b5085488#diff-0f47fd5fac3c82a47477637399052e15c913d0770c8d1bce16f47994f23dd00e

**Review by nickmoreton — APPROVED** [review]
Great thank you.

**Comment by BrianXu20** [comment]
Hi @kevinhowbrook , 
May you take a look and if you are happy, may this be merged?

File: wagtail_guide/tests/settings.py [hunk 2]
```diff
@@ -29,7 +31,7 @@
     'wagtail.documents',
     'wagtail.images',
     'wagtail.admin',
-    'wagtail.core',
+    'wagtail' if WAGTAIL_VERSION >= (3, 0) else "wagtail",
```
Comment (kevinhowbrook) [inline wagtail_guide/tests/settings.py:34]: I missed this... working on a fix

**Pull request merged**